driver-core/src/main/com/mongodb/client/model/vault/RangeOptions.java

```diff
@@ -81,18 +77,21 @@ public RangeOptions max(@Nullable final BsonValue max) {
      * @return the trim factor value if set
      * @since 5.2
      */
+    @Nullable
     public Integer getTrimFactor() {
         return trimFactor;
     }
 
     /**
-     * Set the number of top-level edges stored per record by setting a trim factor, reducing write conflicts during simultaneous inserts
-     * and optimizing queries by excluding seldom-used high-level edges.
+     * Set the number of top-level edges stored per record.
+     * <p>
+     * The trim factor may be used to tune performance.
+     *
      * @param trimFactor the trim factor
      * @return this
      * @since 5.2
      */
-    public RangeOptions setTrimFactor(final Integer trimFactor) {
+    public RangeOptions setTrimFactor(@Nullable final Integer trimFactor) {
```

Comment:
Nit: Can we deprecate this add a `trimFactor` method. Using the API having the `set` prefix really sticks out.

Reply:
Absolutely, good catch! I've renamed it to `trimFactor()`.